### PR TITLE
Tidy up URL parameter section, fix broken crosslink

### PR DESCRIPTION
**Pull request by rschwammborn**

### CODE REVIEW

File: en/elements/basic/map.rst [hunk 1]
```diff
@@ -104,22 +100,18 @@ on the right side. Click on the icon and a popup window will appear.
...
+The second value lists the InstanceID and InstanceLayerId that we want to use now (73-836).
 
-Use this values for the "visibleLayers" parameter in your URL, and seperate them by a slash.
+Use these values for the ``visibleLayers`` parameter in your URL, and seperate them by a slash.
 
 For example: http://localhost/mapbender/application/myapp?visiblelayers=73/836
 
-If you have two layers that are not visible by default, put the two values
-of layerid and serviceid into the URL and seperate them by a comma.
+If you have two layers that are not visible by default, put the two values of layerid and serviceid into the URL and seperate them by a comma.
```
Comment (astroidex) [inline en/elements/basic/map.rst:110]: layerid and serviceid - why not InstanceID and InstanceLayerId

File: de/elements/basic/map.rst [hunk 2]
```diff
@@ -85,40 +85,33 @@ Kontrolle über URL-Parameter
...
 
-Nutzen Sie diese Werte für den "visibleLayers" Parameter in der URL und
-trennen Sie beide Werte mit einem Schrägstrich.
+Nutzen Sie diese Werte für den ``visibleLayers`` Parameter in der URL und trennen Sie beide Werte mit einem Schrägstrich.
 
 Zum Beispiel: http://localhost/mapbender/application/myapp?visiblelayers=73/836
 
-Wenn Sie zwei per Voreinstellung nicht sichtbare Layer haben, fügen Sie
-beide Werte von layerid und serviceid in die URL und trennen diese mit einem Komma.
+Wenn Sie zwei per Voreinstellung nicht sichtbare Layer haben, fügen Sie beide Werte von layerid und serviceid in die URL und trennen diese mit einem Komma.
```
Comment (astroidex) [inline de/elements/basic/map.rst:110]: layerid and serviceid - why not InstanceID and InstanceLayerId

File: en/quickstart.rst [hunk 3]
```diff
@@ -352,7 +352,7 @@ Currently, the element can be used to transfer user- and group information, e.g.
 * toggle allowed: allows opening in the layer tree
 * toggle on: open folder on start of the application
 * more information (...): opens a dialog with detailed layer information
-    * ID: ID of the layer. Can be useful :ref:`to control <layer_visibility>`URL parameters.
+    * ID: ID of the layer. Can be useful :ref:`to control <layer_visibility>` URL parameters.
```
Comment (astroidex) [inline en/quickstart.rst:355]: id or InstanceLayerId?

**Review by astroidex — CHANGES_REQUESTED** [review]

**Review by astroidex — APPROVED** [review]

File: en/quickstart.rst (hunk 3 above)
Comment (astroidex) [reply en/quickstart.rst:355]: @rschwammborn should we change this in the 3.3.x version too?

Comment (rschwammborn) [reply en/quickstart.rst:355]: @astroidex I'm not sure, why should this be necessary? I think it's fine for 3.3.x versions. Instead, we should specify the name of the general id for 4 and ongoing - the current hover text is rather confusing...

layer-id-terminology

**Pull request merged**